app/frontend/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
No big deal but I don't think we are using this file at the moment

Reply:
It was bugging me during accessibility checks on my localhost - big red flag if it's not specified.